publish/samplesite/scripts/controls/sidePane/apiPlayground/insertContent/InsertContentPane.tsx

```diff
@@ -143,6 +147,14 @@ export default class InsertContentPane extends React.Component<
 
     private onClick = () => {
         let editor = this.props.getEditor();
-        editor.addUndoSnapshot(() => editor.insertContent(this.state.content, this.state));
+        if (this.state.position != ContentPosition.Range) {
+            const inputOption: InsertOptionBasic = {
+                position: this.state.position,
+                updateCursor: this.state.updateCursor,
+                replaceSelection: this.state.replaceSelection,
+                insertOnNewLine: this.state.insertOnNewLine,
+            }
```

Comment:
declaring single time use options separately vs inline is kinda inconsistent in this PR. I'd prefe rif it were consistent